src/panels/lovelace/cards/hui-light-card.ts

```diff
@@ -107,7 +90,13 @@ export class HuiLightCard extends LitElement implements LovelaceCard {
           class="more-info"
           @click="${this._handleMoreInfo}"
         ></paper-icon-button>
-        <div id="light"></div>
+        <div id="light">
```

Comment:
```suggestion
        <div id="light">
          ${supportsFeature(stateObj, SUPPORT_BRIGHTNESS) ? html`
          <round-slider
            .value=${brightness}
            @value-changing=${this._dragEvent}
            @value-changed=${this._setBrightness}
          ></round-slider>` : ""}
        </div>
```

Closes https://github.com/home-assistant/home-assistant-polymer/issues/3542

Reply:
Without the slider, how will the card remain any height / be controllable? Probably requires a bit more thought and so better fit for another PR.